src/Compatibility/Core/src/WinUI/RadioButtonRenderer.cs

```diff
@@ -189,7 +189,7 @@ void UpdateFont()
 			if (font == Font.Default && !_fontApplied)
 				return;
 
-			Font fontToApply = font == Font.Default ? Font.SystemFontOfSize(NamedSize.Medium) : font;
+			Font fontToApply = font == Font.Default ? Font.SystemFontOfSize(Device.GetNamedSize(NamedSize.Medium, Element.GetType(), false)) : font;
```

Comment:
Looking at all this ugly `Device.GetNamedSize()` stuff which is in Compatibility... Should we move this to Controls @PureWeen? We need to move everything out of `Device` anyway, so we might as well make this a Controls service now. This is also affected by accessibility things on iOS for font scaling. But I have a sneaky suspicion that this might be for other platforms too.